bs-css/src/Css_AtomicTypes.res

```diff
@@ -1615,18 +1615,18 @@ module TextDecorationThickness = {
 }
 
 module Width = {
-  type t = [#auto | #fitContent | #maxContent | #minContent]
+  type t = [#auto | #fitContent(Length.t) | #maxContent | #minContent]
```

Comment:
This is technically a breaking change. Not entirely sure if it's used anywhere since it was added recently